Gruntfile.js

```diff
@@ -65,6 +66,22 @@ module.exports = function (grunt) {
 		grunt.file.write('test/tmp/3');
 	});
 
+	grunt.registerTask('test4', function () {
+		console.log('test4');
+		grunt.file.write('test/tmp/4');
+	});
+
+	grunt.registerTask('test5', function () {
```

Comment:
I think you can do `var done = this.async()` and do a setTimeout instead of the sleep hack.